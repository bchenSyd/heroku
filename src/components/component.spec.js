import React from 'react';
import renderer from 'react-test-renderer';
import Component from './component';

describe('test component', () => {
    it('render', () => {
        const tree = renderer.create(<Component />);
        expect(tree.toJSON()).toMatchSnapshot();
    }),
    it('render async', async () => {
        const tree = renderer.create(<Component />);
        await Promise.resolve();
        expect(tree.toJSON()).toMatchSnapshot();
    })
})
