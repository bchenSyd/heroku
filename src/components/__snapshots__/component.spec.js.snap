// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component render 1`] = `
<div>
  react component
</div>
`;

exports[`test component render async 1`] = `
<div>
  react component
</div>
`;
